theory-test-admission-dao/src/dao/AdmissionDAO.js

```diff
@@ -58,12 +58,13 @@ class AdmissionDAO extends AbstractDTO {
 
 	static createAdmissionDatabaseRecord(parameters) {
 		const {
-			AdmissionId, DrivingLicenceNumber
+			AdmissionId, DrivingLicenceNumber, HasBooking
 		} = parameters;
 		const admission = new Admission();
 		admission.AdmissionId = AdmissionId;
 		admission.DrivingLicenceNumber = DrivingLicenceNumber;
 		admission.AdmissionStarted = moment(Number.parseInt(new Date().getTime(), 10)).toISOString(true);
```

Comment:
This is a **server** timestamp, not a client timestamp, so you must discard the timezone component.
The reason that you have to explicitly ask for timezone (by passing `true`) is that it's almost always not what you want.

```
/**
 * @returns {string} an ISO-8601 timestamp
 */
function now() {
  return moment().toISOString();
}
```

Reply:
done